py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -245,6 +245,33 @@ def aten_ops_fmod(
     return impl.elementwise.fmod(ctx, target, SourceIR.ATEN, name, args[0], args[1])
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.grid_sampler)  # type: ignore[misc]
```

Comment:
It seems that plain `grid_sampler_2d` and `grid_sampler_3d` are also valid schemas - could those be added here as well?